src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -94,28 +102,112 @@ following to integrate with your `my_flutter` app:
-vary on the pod dependencies of each plugin.
-- A build script is added to the Podfile targets that call `install_all_flutter_pods` to ensure that the binaries you build stay up to date
-with the Dart code that's actually in the folder.  It also uses your Xcode build configuration (Debug, Profile, Release) to embed the matching
-release type of `Flutter.framework` and `App.framework`.
\ No newline at end of file
+- `Flutter.framework` (the Engine library) is being embedded into your application by
+CocoaPods as a vendored framework. This has to match up with the release type
+(debug/profile/release) as well as the architecture for your app (arm*, 
+x86_64, etc.).
+- `App.framework` (your Flutter application binary) is being embedded into your
+application by CocoaPods as a vendored framework.
+- Flutter plugins with iOS platform code are added as CocoaPod pods.
+- A build script is added to the Podfile targets that call `install_all_flutter_pods`
+to ensure that the binaries you build stay up to date with the Dart code in
+the Flutter module.  It also uses your Xcode build configuration (Debug, Profile, Release)
+to embed the matching build mode type of `Flutter.framework` and `App.framework`.
+
+### Embed frameworks in Xcode
+
+Alternatively, you can generate the necessary frameworks and embed them in your application
+by manually editing your existing Xcode project.
+
+If you are using the above [Embed with CocoaPods and Flutter tools](#embed-with-CocoaPods-and-Flutter-tools)
+method, you can skip these instructions.
+
+Let's assume you want to generate the frameworks to `some/path/MyApp/Flutter/`.
+
+```text
+some/path/
+  MyApp/
+    Flutter/
+      Debug/
+        Flutter.framework
+        App.framework
+        FlutterPluginRegistrant.framework
+        example_plugin.framework (each plugin with iOS platform code will be a separate framework)
+      Profile/
+        Flutter.framework
+        App.framework
+        FlutterPluginRegistrant.framework
+        example_plugin.framework
+      Release/
+        Flutter.framework
+        App.framework
+        FlutterPluginRegistrant.framework
+        example_plugin.framework
+```
+
+```terminal
+$ flutter build ios-framework --output=some/path/MyApp/Flutter/
+```
+
+{{site.alert.tip}}
+With Xcode 11 installed you can generate [XCFrameworks][] instead of universal frameworks by adding
+the flags `--xcframework --no-universal`.
+{{site.alert.end}}
+
+Open your existing project in Xcode and drag the frameworks from `some/path/MyApp/Flutter/Release/` in Finder
+into your targets's build settings > General > Frameworks, Libraries, and Embedded Content.
+<div class="container">
+  <div class="row">
+    <div class="col-sm text-center">
+      <figure class="figure">
+        {% asset development/add-to-app/ios/project-setup/embed-xcode.png %}
+        <figcaption class="figure-caption">
+          Embed frameworks in Xcode
+        </figcaption>
+      </figure>
+    </div>
+  </div>
+</div>
+
```

Comment:
while playing, I also noticed that it might not select 'embed & sign' for them automatically. Worth double checking.